src/main/java/org/eclipse/tractusx/puris/backend/common/edc/controller/EdcController.java

```diff
@@ -184,23 +185,6 @@ public ResponseEntity<String> getTransfers(@RequestParam(required = false) Strin
         }
     }
 
-    /**
-     * Get received data from EDC backend application for a specific transfer.
-     *
-     * @param transferId id of the transfer to get data for.
-     * @return response from backend application.
-     */
-    @GetMapping("/backend")
-    @CrossOrigin
-    public ResponseEntity<String> getFromBackend(@RequestParam String transferId) {
```

Comment:
Question: This is not used by existing functionality (response view)?

Reply:
No, that endpoint was used to relay requests to the deprecated Backend Service, which we replaced with the AuthCodesController (now: EndpointDataReferenceReceiver).